src/components/pages/Login.vue

```diff
@@ -13,6 +13,19 @@
           <img src="../../assets/kitsu-text.svg" v-else />
         </div>
         <form v-if="!(isMissingOTP || isWrongOTP)">
+          <div class="field" v-if="mainConfig?.saml_enabled">
+            <p class="control">
+              <a
+                class="button is-fullwidth"
+                :class="{
+                  'is-loading': isLoginLoading
+                }"
+                @click="loginSAML"
```

Comment:
You can use a native link directly instead of a click event:

```suggestion
                href="/api/auth/saml/login"
```